### PR TITLE
Eip 123 GitHub actions for FE

### PR DESCRIPTION
**Pull request by RenanDbs**
Multiple commit are due to the obligation to push each time i want to test it

### CODE REVIEW

File: package-lock.json
Comment (Alban-de-tourtier) [inline package-lock.json:1]: Remove the package-lock.json

File: .github/workflows/ci_fe.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: Node.js CI
```
Comment (JosefineM2000) [inline .github/workflows/ci_fe.yml:4]: Please rename to something with FE, for example "Frontend CI"

**Review by JosefineM2000 — CHANGES_REQUESTED** [review]

File: .github/workflows/ci_fe.yml (hunk 1 above)
Comment (RenanDbs) [reply .github/workflows/ci_fe.yml]: Done and changed to Frontend VisitorWeb CI

**Review by JosefineM2000 — APPROVED** [review]
Pay attention to the merge conflict, otherwise LGTM

**Pull request merged**